modules/burials/app/models/burials/saved_claim.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
```

Comment:
Adjusted namespace for module. Since they both inherit from `SsavedClaim` this is an instance of STI (Single Table Inheritance) so as long as we don't do lookups in any of the jobs outside of `SavedClaim.find` and not prefix it we are good.

Rest of the logic has not changed